skycultures/anutan/info.ini

```diff
@@ -1,3 +1,4 @@
 [info]
 name = Anutan
 classification = ethnographic
+license = CC BY-SA 4.0 International Public Licence
```

Comment:
why do we name the option "license" (American standard programmer language) and then often, but not everywhere write "licence" (the "very British" format) while the CC licenses are commonly written again in American English, "license" ?

Reply:
"Licence" this is typo

Reply:
OK, done!